js/libs/core.js

```diff
@@ -883,6 +888,63 @@ Fliplet.FormBuilder = (function() {
         component.methods.openFilePicker = component.methods._openFilePicker;
       }
 
+      component.methods._getFieldOptions = function() {
+        var $vm = this;
+        var fields = $vm.$parent.fields;
+
+        if ($vm.fieldOptions.length < 1) {
```

Comment:
@armine-fliplet, that's a pretty mysterious expression. There's only one option, `length === 0`. And since 0 is falsy, you can use it like this:

```suggestion
        if (!this.fieldOptions.length) {
```